src/agni/server/dataguard/I_UserDataGuard.java

```diff
@@ -19,4 +19,8 @@
     public void changeUserCurrentChat(String user, String chatname);
 
     public String userCurrentChat(String user);
+
+    public void userLogin(String ip, String user);
```

Comment:
I think `loginUser` is  a better name as it's describing the action we are taking.